### ISSUE TITLE
Missing files in report

### ISSUE THREAD

**Issue by steveriggins** [SlatherOrg/slather#214]
Hi!

I am building a fairly complex project with many frameworks in Xcode 7. I have set up Slather to output to teamcity, but it was showing a rather low coverage number.  I then ran Slather manually, this time outputting to HTML:

slather coverage --html --show

The resulting html directory has a ton of html files in it, but the generated index.html is only showing a subset, and no hierarchy of files that match the hierarchy in the project.

Is this a known issue, or something I am missing?

Thank you!

**Comment by sbhklr** [comment]
@steveriggins Did you specify the source files using `--source-directory`?

**Comment by steveriggins** [comment]
No, I let it cover the entire project directory. I did find that can search for any file, and it will be found, but only the first 100 or so are shown on the index page, and there are no folders for showing group %, and then per file %, etc.

**Comment by sbhklr** [comment]
Make sure that the file system representation reflects your project's hierarchy. As far as I know slather doesn't process the .xcproj for generating the hierarchy. Can you provide us with a demo project to see what's going on?

**Comment by devikirankhode** [comment]
Have you solved it by any chance @steveriggins ? I am facing the same problem now.

**Comment by devikirankhode** [comment]
The command that i used here is:
`slather coverage --scheme "Myscheme" --html --output-directory "coverage/reports" --workspace Myscheme.xcworkspace --input-format profdata --build-directory "/Users/xxxx/Library/Developer/Xcode/DerivedData" --source-directory . Mysheme.xcodeproj`

It gave 23% coverage,but on Xcode it's showing 78%

**Comment by sbhklr** [comment]
Can you provide us with a sample project to reproduce the issue?

**Comment by calvincestari** [comment]
Can confirm I'm having the same issue; the paging is broken in ```list.min.js```.

I don't care for a page limit so I raised the ```k.page = 200``` value to 200000 and I can now see all files. I'm useless at Javascript so I've got no idea how to actually fix it.

**Comment by ehlersd** [comment]
Any chance of getting this issue resolved?

**Comment by sbhklr** [comment]
@ehlersd I'll have a look at it if someone provides me with a sample project.

**Comment by MacGritsch** [comment]
Same here. And as I am totally new to `gem` I dont know how I can patch it.
I found the problem in the javascript file and fixed it. But how can I build and run slather now?
Thank you very much!

**Comment by ksuther** [comment]
Here's how to build and install slather:

`gem build slather.gemspec ; gem install slather-2.4.3.gem`

**Comment by MacGritsch** [comment]
Thank you very much @ksuther!
I had to execute `export LC_ALL=en_US.UTF-8` first (otherwise build stops because of invalid character).
